Library/Formula/premake5.rb

```diff
@@ -0,0 +1,18 @@
+class Premake5 < Formula
```

Comment:
No idea. Don't use it enough to know the answer to that question. Essentially, whichever is going to be less useful *for now* should live in Versions.